rviz_visual_testing_framework/src/visual_test_fixture.cpp

```diff
@@ -38,13 +38,17 @@
 void VisualTestFixture::SetUpTestCase()
 {
   QLocale::setDefault(QLocale::English);
-  int argc = 0;
+  int argc = 1;
+  char * argv[] = {const_cast<char *>("rviz2"), nullptr};
```

Comment:
This is scary, since `QApplication()` might try to modify it.

Reply:
We have removed the const_cast and now use directly a char array.